### ISSUE TITLE
[binutils] Man pages are empty

### ISSUE THREAD

**Issue by Alto1772** [msys2/MSYS2-packages#2946]
## Describe the issue

The binutils package doesn't contain any program's manpage at all. They're just gzipped files with nothing on it...

## Steps to Reproduce the Problem

1. Install 'binutils'
2. Run `man nm` (or any program eg. 'objdump', 'ld', 'as')
3. It outputs just blank empty.
4. I expected it to display a manpage of nm(1).

## Additional Context:

There is an available patch that can be found in here: https://github.com/freebsd/freebsd-ports/commit/a055899f4f1c41d07ae5b8d3f5a9983173ef76f6

![mintty_Mr9YcSZyoP](https://user-images.githubusercontent.com/56553686/164911802-75ca36a8-063a-4d96-8458-2abbec34ee56.png)

**Comment by Biswa96** [comment]
Is this issue fixed in binutils 2.38? The mingw binutils shows man pages correctly.

**Comment by Alto1772** [comment]
Yes, i think so, but will this msys package get updated to 2.38 yet?

**Comment by lazka** [comment]
(we are at 2.46.1 across the board now)
